cloudinary-core/src/main/java/com/cloudinary/Configuration.java

```diff
@@ -87,11 +89,12 @@ private Configuration(
         this.proxyPort = proxyPort;
         this.secureCdnSubdomain = secureCdnSubdomain;
         this.useRootPath = useRootPath;
-        this.timeout = 0;
+        this.timeout = timeout;
```

Comment:
Why was that changed?

Reply:
It's a bugfix (will be in the commit message on merge).